cetmix_tower_server/readme/HISTORY.md

```
@@ -1,3 +1,8 @@
+## 16.0.2.2.5 (2025-12-10)
+
+- Bugfixes: Custom values in flight plan are lost in a skipped command and are not available after it. (5129)
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)